.travis.yml

```diff
@@ -5,6 +5,7 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 install:
+  - npm install bitcore
```

Comment:
Actually after a second check, it's necessary. It doesn't look like peerDependencies behave similarly to devDependencies.